### PR TITLE
Added Custom Config Path option

### PR DESCRIPTION
**Pull request by dacodekid**
Added Custom Config Path option & fixed double quote to single quote for code consistency.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -95,7 +104,8 @@ export default class LinterJSCS {
         this.jscs.registerDefaultRules();
 
         const filePath = editor.getPath();
-        const config = configFile.load(false, path.dirname(filePath));
+        const config = configFile.load(false,
+          path.join(path.dirname(filePath), this.configPath));
```
Comment (Arcanemagus) [inline index.js]: You could also just use [this](https://github.com/AtomLinter/atom-linter/blob/master/lib/helpers.coffee#L121), which would search the current directory and any parents.

**Comment by donaldpipowitch** [comment]
Any update on this? :)

**Comment by Arcanemagus** [comment]
This seems to work as is, there are several things that could be improved other than just the comment above (like observing the settings instead of `get()`ing them every time), so I'll just merge this in for anyone using this package.

**Comment by Arcanemagus** [comment]
Published in v3.2.0.

**Pull request merged**